templates/ApiGear/Source/ApiGear/Private/unrealolink.cpp

```diff
@@ -55,7 +56,7 @@ UUnrealOLink::UUnrealOLink(const FObjectInitializer& ObjectInitializer)
 		}
 		processMessages();
 	};
-	m_node.onWrite(func);
+	m_node->onWrite(func);
 
```

Comment:
not able to add comment to dtor
if the node owner dies before sink is unlinked, the dtor should remove node for all sinks from the registry, you can check asking the registry for objects for this node (before the dtor of node is called, casue after that the registry will always return nullptr on storing node lock - this is why it coudn't be done in dtor of node)

Reply:
should also be solved. When the node is destroyed(deinit for the subsystem) we disable the callback functions